.gitignore

```
@@ -39,6 +39,8 @@ yarn-error.log
 # Misc
 .DS_Store
 *.pem
+.idea
+.vscode
```

```suggestion
.idea
```
Keeping `.vscode` for recommended vscode extensions 🙂